android/src/fdroid/play/release-notes/en-US/default.txt

```diff
@@ -1,11 +1,13 @@
-• New OSM data as of Feb 4, 2022
-• Fixed a broken place page on Android 11
-• Fixed invalid routes for car&bike
-• Respect "Avoid" setting for bike/walking routes
-• No unlock is needed when OM is active
-• Search for Water, RV Facilities and Recycling
-• Added parcel pickup, excrement bags, quay, marina, caravan, dump station, mast, flag pole, funeral directors
-• Improved translations, FAQ in Turkish
-• Fixed HTML display in bookmark's description
-
-…see https://organicmaps.app for a full changelog!
+• New OSM data as of March 14, 2022
+• Wikipedia articles for well-known places
+• Search for organic shops and restaurants
+• Material style for bottom menus and "show on map"
+• Filter inaccurate network locations
+• Fixed compass overlapped by layers button for RTL
+• Tuned visibility of subway entrances, steps, footways, community_centre
+• Brighter roads in dark mode
+• Darker farmlands and parkings
```

Comment:
Display parkings, parks, etc. over hospital grounds
#2166

Reply:
There is a 500 chars limit for Android what's new.

Reply:
Yes, but hospital change is more important than darker farmlands (which is a part of "Tuned visibility of steps, footways"), imho